### DISCUSSION TITLE
Button to Apply start and end of sync

### DISCUSSION THREAD

**Discussion by ElisameAraujo** [SubtitleEdit/subtitleedit#7231] [Ideas]
I have a suggestion for the possibility of automatic subtitle synchronization: A button that, when clicked, started subtitle synchronization and ended as soon as it was released.

The program I still use today that has this function is DivX Land Media Subtitler - the project was discontinued in 2017. It has some limitations that Subtitle Edit does not have, such as support for texts in ANSI.

Is it possible to add something similar to Subtitler Edit?

In the attached print there is the detail of the synchronization button and, in the box where the text is, it is possible to see the coding problem.

![DivXLand Media Subtitler](https://github.com/SubtitleEdit/subtitleedit/assets/40779799/1c408e8f-fab7-4c7b-aa1a-1b96fad13878)

**Comment by niksedk** [comment]
Something like this?

![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/c4ba760a-e661-4364-969f-51ef49b03c3d)

**Comment by ElisameAraujo** [comment]
Yep, something like that, but in a slightly more visual way - more explicit -, because it's not something you'll find right away in the program if you need to synchronize a subtitle, at least that's what I see, because I do a lot of subtitles.

**Comment by niksedk** [comment]
It's very visual using the waveform... give it a try.

Could it be better? What do you mean by more visual?

**Reply by ElisameAraujo** [reply]
I did a quick test with the shortcut, it works relatively well, when I went to see the result I felt a small delay at the beginning of the subtitle entry, but nothing too exaggerated. Perhaps a delay between the pressed button and the actual action in the program.

When I mean visual, I literally mean it. Having an option you can do this with the click of the mouse, which I believe has no delay.

If you need it, I have the DivXLand installer here for you to have an idea or I can send you a video showing how it works.
